apps/editors/views.py

```diff
@@ -183,25 +191,30 @@ def home(request):
         durations=durations,
         reviews_max_display=reviews_max_display,
         motd_editable=motd_editable,
-        queue_counts_total=queue_counts(admin_reviewer=True),
+        queue_counts_total=queue_counts(admin_reviewer=True,
+                                        limited_reviewer=limited_reviewer),
         unlisted_queue_counts_total=queue_counts(admin_reviewer=True,
-                                                 unlisted=True),
+                                                 unlisted=True,
+                                                 limited_reviewer=limited_reviewer),
```

Comment:
It's saying that the line is longer than 79 characters which is too long (mainly we want to shorten line length so that people can use split view efficiently). You just need to make the line shorter. One easy way is to switch this from a vertical indent (that's what it is now) to a hanging indent, which is:
````
unlisted_queue_counts_total=queue_counts(
    admin_reviewer=True,
    unlisted=True, ...)
````